### ISSUE TITLE
Extract individual flows from pcap

### ISSUE THREAD

**Issue by A1337CBS** [chmorgan/sharppcap#148]
Hi, I was wondering if it was possible to extract individual flows from a pcap file as you do in wireshark by applying the filter "tcp.steam == 1" or "udp.stream==1"? I tried applying it on the device filter but it gave me an error. Note that I am reading an already captured pcap file. 

Thank you.

**Comment by chmorgan** [comment]
Hello!

Aren’t the filters the same for online and offline filters? Can you check
the libpcap docs to confirm.

**Comment by A1337CBS** [comment]
Hi @chmorgan, I thought they are the same filters and I checked the docs but I guess its probably only a wireshark filter then? Anyway let me rephrase the question to just flows, using Sharppcap how would you extract flows or streams from a pcap file?

**Comment by chmorgan** [comment]
Hi @A1337CBS. You should be able to use packetdotnet-connections, https://github.com/chmorgan/packetnet-connections, to track and follow that data. I developed that for another project, I'm guessing it could be greatly improved with more usage and development.

**Comment by chmorgan** [comment]
Closing out as packetnet-connections could be an option here in addition to using a bpf.
